README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://travis-ci.org/dbtsai/python-mimeparse.svg?branch=master
+   :target: https://travis-ci.org/dbtsai/python-mimeparse
+
+This module provides basic functions for handling mime-types. It can
+handle matching mime-types against a list of media-ranges. See section
+5.3.2 of the HTTP 1.1 Semantics and Content specification [RFC 7231] for
+a complete explanation: https://tools.ietf.org/html/rfc7231#section-5.3.2
+
+Functions
+=========
+
+``parse_mime_type()``
```

Comment:
I think we could use smaller headers for these function names. It looks pretty ugly right now:

<img width="1015" alt="screen shot 2016-07-08 at 4 28 46 pm" src="https://cloud.githubusercontent.com/assets/1718372/16704192/2e341be0-4529-11e6-9208-a57bdf328c2b.png">